File: isa-mrs-2022/Spring-Boot-Backend/backend/src/main/java/rs/ac/uns/ftn/backend/dto/response/AdventureReservationCalendarDTO.java
package rs.ac.uns.ftn.backend.dto.response;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import rs.ac.uns.ftn.backend.dto.request.SimpleActionDTO;
import rs.ac.uns.ftn.backend.model.Adventure;
import rs.ac.uns.ftn.backend.model.AdventureReservation;
import rs.ac.uns.ftn.backend.model.BoatResevation;

import java.util.ArrayList;
import java.util.List;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class AdventureReservationCalendarDTO {
    private List<AdventureReservation> adventureResevations= new ArrayList<>();

    private List<SimpleActionDTO> adventureActionsReservated= new ArrayList<>();

    private List<SimpleActionDTO> adventureActionsUnReservated= new ArrayList<>();
}

File: isa-mrs-2022/Angular-Frontend/angular-frontend/src/app/models/response/http-adventure-response/AdventureReservationCalendarDTO.ts
import { SimpleActionDTO } from "../../util/SimpleActionDTO";
import { AdventureResevationDTO } from "./adventure-reservation";

export class AdventureReservationCalendarDTO {

    adventureResevations: AdventureResevationDTO[];

    adventureActionsReservated: SimpleActionDTO[];

    adventureActionsUnReservated: SimpleActionDTO[];

}